Header header
vision_msgs/BoundingBox3D bounding_box_3D
float32 probability
